components/generate/layer/steps/CreateLayeredTicketStep4.tsx

```diff
@@ -39,7 +41,7 @@ function CreateLayeredTicketStep4() {
           layer.layerOccurrence = data.layers[i].layerOccurrence
           layer.layerName = data.layers[i].layerName
           layer.assets.forEach((asset, j) => {
-            (asset.occurrence = data.layers[i].assets[j].occurrence),
+            ;(asset.occurrence = data.layers[i].assets[j].occurrence),
```

Comment:
wild semi-colon appeared

Reply:
husky said no! (some format weird error in VSCode) but it has no error